### ISSUE TITLE
Using <abbreviated-form> throws [DOTX060W] (key is not associated with a glossary entry) and no output is generated

### ISSUE THREAD

**Issue by xephon2** [dita-ot/dita-ot#2762]
## Expected Behavior
The `<abbreviated-form>` element should be rendered correctly and show the value of the `<glossSurfaceForm>` form element of the glossary entry.

## Actual Behavior
The **[DOTX060W]** error is thrown in the **transform.topic2fo.main** phase and the `<abbreviated-form>` is not rendered.

## Possible Solution
The processing of `<glossentry>` topics was incorrect for HTML-based transformations before, but now seem to be broken for **pdf2** as well. I'd recommend to fix this issue in parallel with #2173.

## Steps to Reproduce

**map.ditamap**
```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE map PUBLIC "-//OASIS//DTD DITA Map//EN" "map.dtd">
<map>
  <title>map</title>
  <topicref href="topic.dita"/>
  <mapref href="glossary.ditamap"/>
</map>
```

**glossary.ditamap**
```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE map PUBLIC "-//OASIS//DTD DITA Map//EN" "map.dtd">
<map>
  <title>glossary</title>
  <glossref keys="foo" href="foo.dita"/>
</map>
```

**topic.dita**
```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE topic PUBLIC "-//OASIS//DTD DITA Topic//EN" "topic.dtd">
<topic id="topic">
    <title>topic</title>
    <body>
        <p>Lorem Ipsum <abbreviated-form keyref="foo"/>.</p>
    </body>
</topic>
```

**foo.dita**
```xml
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE glossentry PUBLIC "-//OASIS//DTD DITA Glossary//EN" "glossary.dtd">
<glossentry id="foo">
  <glossterm>foo</glossterm>
  <glossdef>just foo</glossdef>
  <glossBody>
    <glossSurfaceForm>foo (f)</glossSurfaceForm>
    <glossAlt>
      <glossAcronym>f</glossAcronym>
    </glossAlt>
  </glossBody>
</glossentry>
```

[glossary-test.zip](https://github.com/dita-ot/dita-ot/files/1199994/glossary-test.zip)

1. Download and extract the test files.
2. Run the following command in the `glossary-test` directory.
   ```
   $ dita --input map.ditamap --format pdf2
   ```

## Copy of the error message, log file or stack trace
[dita.log.zip](https://github.com/dita-ot/dita-ot/files/1200010/dita.log.zip)

## Environment

* DITA-OT version: **2.5.2**
* Operating system and version: **Windows 7**
* How did you run DITA-OT? **dita command**
* Transformation type: **pdf2**

**Comment by xephon2** [comment]
Duplicate of #2773
